app/charts/combo/combo-line-single-state-props.ts

```diff
@@ -47,14 +47,17 @@ export const useComboLineSingleStateVariables = (
     { dimensionsById }
   );
 
+  const unitConversionFactor = y.unitConversion?.factor ?? 1;
+
   const numericalYVariables: NumericalYComboLineSingleVariables = {
     y: {
       lines: chartConfig.fields.y.componentIds.map((id) => ({
         dimension: measuresById[id],
         id,
         label: measuresById[id].label,
         color: fields.color.colorMapping[id],
-        getY: (d) => (d[id] !== null ? Number(d[id]) : null),
+        getY: (d) =>
```

Comment:
I think we should use a function convertUnit(Number(d[id], y.unitConversion) here.

1. The state would not have to know about how to multiply and about this factot property
2. We would reduce duplication of this knowledge everywhere
3. We could have in the future unit conversion which is not linear (this happens for various unit pairs)